src/modules/toast.js

```diff
@@ -0,0 +1,10 @@
+const showToast = (message) => {
+  const toast = document.getElementById('toast');
+  toast.innerText = message;
+  toast.style.display = 'block';
+  setTimeout(() => {
+    toast.style.display = '';
+  }, 3000);
+};
+
+module.exports = showToast;
```

Comment:
I loved this feature, kudos for this! 🚀 

- [ ] **[OPTIONAL]** The same happens here, since you're using ES6 syntax, please consider using import/exports instead of commonJS's `module.exports`, I'm marking this as optional since this feature isn't part of the requirements, however, please be consistent with the ES6 syntax! 🤓 🔧 


```suggestion
export const showToast = (message) => {
  const toast = document.getElementById('toast');
  toast.innerText = message;
  toast.style.display = 'block';
  setTimeout(() => {
    toast.style.display = '';
  }, 3000);
};


```